inventory/src/main/java/org/flyve/inventory/categories/Networks.java

```diff
@@ -348,25 +353,53 @@ public String getDescription() {
 		return name;
 	}
 
-	public String getLocalIpV6() {
+	public String getAddressIpV6() {
+		InterfaceAddress interfaceIPV6 = getInterfaceByType("IPV6");
+		if (interfaceIPV6 != null) {
+			String address = interfaceIPV6.getAddress().getHostAddress();
+			return address.contains("%") ? address.split("%", 2)[0] : "N/A";
+		} else {
+			return "N/A";
+		}
+	}
+
+	public String getMaskIpV6() {
+		InterfaceAddress interfaceIPV6 = getInterfaceByType("IPV6");
+		if (interfaceIPV6 != null) {
+			String address = interfaceIPV6.getAddress().getHostAddress();
+			return address.contains("%") ? address.split("%", 2)[0] : "N/A";
+		} else {
+			return "N/A";
+		}
+	}
+
+	/** Get interface address
+	 * @param type IPV6 or IPV4
+	 * @return interface address
+	 */
```

Comment:
If you are adding documentation you need to check the proper checkbox in the PR body 😉

Reply:
Ok 👍